### PR TITLE
Beta release

### PR DESCRIPTION
**Pull request by switowski**
Waiting for https://github.com/inveniosoftware/invenio-search/pull/111 to be merged.

### CODE REVIEW

**Comment by switowski** [comment]
Ready to be reviewed and merged.

File: CHANGES.rst [hunk 1]
```diff
@@ -24,7 +24,7 @@
 Changes
 =======
 
-Version 1.0.0a11 (released 2017-10-16)
+Version 1.0.0b1 (released 2017-10-23)
 --------------------------------------
 
 Incompatible changes
```
Comment (drjova) [inline CHANGES.rst]: Leaving incompatible changes for beta release is a bit weird, can we rephrase to something like [Initial public release](https://github.com/inveniosoftware/invenio-oauthclient/blob/master/CHANGES.rst#version-100b2-released-2017-08-10)? WDYT? :)

Comment (switowski) [reply CHANGES.rst]: Good point, I've updated it.

**Comment by drjova** [comment]
One more thing :) ``RELEASE-NOTES.rst`` seems that is missing. In all the other repos we have it, wasn't it part of the release checklist?

**Comment by switowski** [comment]
We decided to remove `RELEASE-NOTES.rst`. It's actually mentioned in the revised checklist, that I linked to in HackMD :)

**Pull request merged**